crates/bevy_core_pipeline/src/deferred/node.rs

```diff
@@ -122,17 +122,17 @@ impl ViewNode for DeferredGBufferPrepassNode {
         let view_entity = graph.view_entity();
         render_context.add_command_buffer_generation_task(move |render_device| {
             #[cfg(feature = "trace")]
-            let _deferred_span = info_span!("deferred").entered();
+            let _deferred_span = info_span!("deferred_prepass").entered();
```

Comment:
Why the rename to add `_prepass` everywhere?

Reply:
It was a bit confusing seeing "deferred" in the profiler. This made it a bit clearer, and matched the node name.